.wordlist.txt

```diff
@@ -103,3 +103,9 @@ DGEMM
 HPCG
 HPL
 IOPM
+amdclang
+asan
+complilation
```

Comment:
compilation, this line shouldn't be added

Reply:
Agreed. Please remove 'compilation'.

Reply:
You need to remove the line.